packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/VirtualizedTraceView.tsx

```diff
@@ -220,6 +223,18 @@ export class VirtualizedTraceViewImpl extends React.Component<VirtualizedTraceVi
     }
   }
 
+  focusSpan = (uiFind: string) => {
```

Comment:
actually I don't think this will work due to these calls synchronously updating the redux state but the render that would consume the change caused by `focusUiFindMatches` is async... so I think this is fine for now